test/images/ipc-utils/README.md

```diff
@@ -0,0 +1,13 @@
+# ipc-utils
+
+This container will be used to exercise the HostIPC functionality in
+e2e-node tests.
+
+The version of `ipcs` shipped in busybox performs operations that get
+blocked by SELinux on hosts where it is enabled. The version of `ipcs`
+in util-linux does not perform those operations, rather it checks
+whether the /proc files it needs are available and proceeds to reading
```

Comment:
proceeds to read* ?

Reply:
👍 I just copy/pasted from the original PR that added this image (just for context)